lib/datura/elasticsearch/index.rb

```diff
@@ -226,7 +229,8 @@ def self.clear_index(options)
 
     if confirmation
       data = self.build_clear_data(options)
-      RestClient.post(url, data.to_json, { content_type: :json }) { |res, req, result|
+      auth_header = Datura::Helpers.construct_auth_header(options)
```

Comment:
Note for future reference on committing. This helper method isn't included in this commit also, so the code would be broken at this commit. It's added in the next commit, but in the future these kinds of things need to be committed together.